### PR TITLE
Add enable_mesos_input_plugin config option

### PR DESCRIPTION
**Pull request by gracedo**
https://jira.mesosphere.com/browse/DCOS-47301
Adds the enable_mesos_input_plugin config option.

Terraform Core PR: dcos-terraform/terraform-template-dcos-core#2
DCOS PR: dcos/dcos#4222

### CODE REVIEW

File: main.tf
Comment (geekbass) [inline main.tf:64]: Please set this source back prior to merge.

File: main.tf
Comment (geekbass) [inline main.tf]: Please set version back prior to merge.

**Review by geekbass — CHANGES_REQUESTED** [review]
Code LGTM outside of needing to update the variable name and description. Please update to:

dcos_enable_mesos_input_plugin with description "Indicates whether to enable Telegraf's Mesos input plugin to collect Mesos metrics from Mesos masters and agents. Options: `true` or `false` (optional)"

Adding @fatz as second pair of eyes.

**Review by bernadinm — APPROVED** [review]
LGTM!
When Jenkins is back up, let's rerun the job before merging if possible.

**Review by geekbass — APPROVED** [review]

**Comment by bernadinm** [comment]
manually ran the test. Here is the output, good to merge:
```
$ terraform fmt
$ terraform validate --check-variables=false
$ docker run -ti -v$(pwd):/a --entrypoint /usr/bin/terraform-docs fatz/terraform-jenkins md /a | tr -d '\r' > README.md
$ git status
On branch gracedo/enable_mesos_input_plugin_config_option
Your branch is up to date with 'origin/gracedo/enable_mesos_input_plugin_config_option'.```

**Pull request merged**
